tests/functional/cylc-trigger/08-edit-run-host-select/suite.rc

```diff
@@ -13,8 +13,8 @@ second task fixes and retriggers it with an edit-run."""
 [runtime]
     [[broken-task]]
         script = false
-        [[[remote]]]
-            host = $(echo localhost)
+        # TODO: review comment: was this testing host select type logic
+        # if so should it be converted to platform select type logic?
```

Comment:
At the minute we cannot specify `platform = $(echo localhost)` (or at least I don't think we can) so leave as a todo until this is reimplemented?